src/test/java/io/cdap/plugin/gcp/dataplex/source/config/DataplexBatchSourceConfigTest.java

```diff
@@ -86,7 +86,14 @@ public void testValidateTable() {
       dataplexBatchSourceConfig.validateBigQueryDataset(mockFailureCollector, "project", "dataset", "table-wrong");
     } catch (Exception e) {
     }
-    assertEquals(2, mockFailureCollector.getValidationFailures().size());
+
+    assertTrue("Expected at least one error with incorrect table name",
+               mockFailureCollector.getValidationFailures().size() > 0);
+
+    mockFailureCollector.getValidationFailures().stream()
+      .filter(error -> error.getFullMessage().toLowerCase().contains("table name"))
```

Comment:
should we just check if it contains 'table-wrong' in the error message?

Reply:
yeah we could do that, but it would need to be a separate PR on develop first. This is a cherry-pick, so I don't want to make changes here that are not in develop

Reply:
got it.